spec/replication/replication_spec.rb

```diff
@@ -0,0 +1,166 @@
+require_relative "./replication_helper"
+
+describe "pglogical replication" do
+  let(:slave_db_name)  { "#{ActiveRecord::Base.connection.current_database}" }
+  let(:master_db_name) { "#{slave_db_name}_master" }
+  let(:rep_set_name)   { "test_rep_set" }
+  let(:sub_name)       { "test_subscription" }
+
+  let(:excluded_tables_regex) do
+    excluded_tables = @replication_config[:exclude_tables].join("|")
+    excluded_tables = "^(#{excluded_tables})$"
+    Regexp.new(excluded_tables)
+  end
+
+  let(:conn_info) do
+    config = VMDB::Config.new("database").config[:test]
+    dsn = ""
+    dsn << "user=#{config[:username]} " if config[:username]
+    dsn << "password=#{config[:password]} " if config[:password]
+    dsn << "host=#{config[:host]}" if config[:host]
+    dsn
+  end
+
+  let(:slave_dsn)  { "dbname=#{slave_db_name} #{conn_info}" }
+  let(:master_dsn) { "dbname=#{master_db_name} #{conn_info}" }
+
+  before do
+    skip "pglogical must be installed" unless ActiveRecord::Base.connection.pglogical_installed?
+
+    MiqRegion.seed
+    Zone.seed
+    MiqServer.seed
+
+    master_db_config = VMDB::Config.new("database").config[:test].merge(:database => master_db_name)
```

Comment:
if you need the keys symbolized (to store it in the config below, for example), just do `Rails.configuration.database_configuration[Rails.env].symbolize_keys`